RELEASE_NOTES.md

```diff
@@ -11,6 +11,17 @@
 ## Added
 - `showPreselectedStoredPaymentMethod` flag to `DropInConfiguration` to allow choosing to skip to the payment method selection screen.
 - Any `Configuration` will check if the `clientKey` matches the `environment` it will be used on.
+- Postal Code input field to Card Component.
+- KCP Authentication input field support to Card Component.
+- Social Security Number (CPF/CNPJ) input field support to Card Component.
+- Android 12 support.
+
+## Changed
+- Visibility of Expiry Date in Card Component changed to an enum with REQUIRED, OPTIONAL and HIDDEN as options.
```

Comment:
This is an internal change and doesn't have any effect for the merchant so no need to add here.

Reply:
removed.